src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/JsonConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.config;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.main.JsonSchema;
+import com.github.fge.jsonschema.main.JsonSchemaFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@Configuration
+public class JsonConfiguration {
+
+    @Bean
+    public ObjectMapper defaultObjectMapper() {
+        ObjectMapper mapper = new ObjectMapper();
+
+        mapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
+
+        return mapper;
+    }
+
+    @Bean
+    public JsonSchema jsonSchemaValidator(ObjectMapper mapper) throws IOException, ProcessingException {
```

Comment:
Worth clarifying by changing method name or adding a javadoc IMHO - this is not just general json schema validator.
It's a validator for a very specific metadata file schema for metadata file.

Also if this bean is used only once, perhaps the `JsonSchema` object can be initialized inside the class that uses it?